content/docs/tasks/telemetry/kiali/index.md

```diff
@@ -0,0 +1,345 @@
+---
+title: Visualizing Your Mesh
+description: This task shows you how to visualize your services within an Istio mesh.
+weight: 50
+keywords: [telemetry,visualization]
+---
+
+This task shows you how to visualize different aspects of your Istio mesh.
+
+As part of this task, you will install the [Kiali](https://www.kiali.io) add-on and use
+the web-based graphical user interface for viewing service graphs of the mesh
+as well as your Istio configuration objects such as virtual services
+and destination rules. You will also use the Kiali API to generate graph data
+in the form of consumable JSON.
+
+The [Bookinfo](/docs/examples/bookinfo/) sample application is used as
+the example application throughout this task.
+
+## Before you begin
+
+{{< idea_icon >}} If you already have Istio installed and need Kiali installed, you can follow the [Kiali install instructions here](https://www.kiali.io/gettingstarted/).
+
+Before you install [Istio](/docs/setup/) in your cluster, you first need to create a secret in your Istio namespace with credentials that users will use to authenticate to Kiali. See the [Helm README](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/README.md#installing-the-chart) for details, but here is a command you can run to create a secret:
+
+{{< text ascii >}}
```

Comment:
`{{< text yaml >}}` or `{{<text bash>}}` should render better.

Reply:
@sdake I did use text bash before, and I can switch back to using it again, but it doesn't render as nice as you'd think. It thinks the "here document" is part of the output:

![image](https://user-images.githubusercontent.com/2029470/46369284-807a9280-c650-11e8-833e-54663422baaf.png)

So I can't click the "copy" button and have the full set of commands copied to the clipboard nicely. The way it is now (with text ascii), I can click the "copy" button, go over to my console window, "paste" and the command executes and everything is created on one shot.

Does anyone know how to get this markdown "test bash" to understand the "here document" ?

Reply:
FYI: I switched it to use "```bash" and it renders it better while retaining the ability copy-n-paste into a console window.